posthog/models/feature_flag/flag_matching.py

```diff
@@ -138,14 +138,12 @@ def get_match(self, feature_flag: FeatureFlag) -> FeatureFlagMatch:
 
         # Match for boolean super condition first
         if feature_flag.filters.get("super_groups", None):
-            super_condition_value_is_set = self._super_condition_is_set(feature_flag)
-            super_condition_value = self._super_condition_matches(feature_flag)
-
-            if super_condition_value_is_set:
+            is_match, super_condition_value, evaluation_reason = self.is_super_condition_match(feature_flag)
```

Comment:
really think we should incorporate this into `is_condition_match`, because we want all of the property overrides handling in here too.

Reply:
.. unless I'm missing something here which makes this much harder to do

Reply:
the property overrides represent the most up to date state of the end user though: If the property updates haven't been ingested, it's these overrides that ensure the experience for the end user is consistent.

Reply:
ah fair point